frontend/src/components/files/FileActionsMenu.tsx

```diff
@@ -74,8 +77,19 @@ export const FileActionsMenu = (props: FileActionsMenuProps): JSX.Element => {
 	const history = useNavigate();
 
 	const [confirmationOpen, setConfirmationOpen] = useState(false);
-
 	const [updateFileOpen, setUpdateFileOpen] = useState(false);
+	const [shareEnabled, setShareEnabled] = useState(false);
+
+	useEffect(() => {
+		if (storageType === "minio") {
+			console.log("share OK");
```

Comment:
Sorry nitpicking, could you remove console log?

Reply:
done